### PR TITLE
Update default values for XmlReaderSettings

### PR DESCRIPTION
**Pull request by omajid**
- Sort the table alphabetically
- Add missing `CloseInput` property
- Fix link to `DtdProcessing` property: it should point to System.Xml.XmlReaderSettings.DtdProcessing.

### CODE REVIEW

File: xml/System.Xml/XmlReaderSettings.xml [hunk 1]
```diff
@@ -123,7 +123,9 @@ The following example creates an <xref:System.Xml.XmlReader> that uses an <xref:
 |--------------|-------------------|
 |<xref:System.Xml.XmlReaderSettings.Async%2A>|`false`.|
 |<xref:System.Xml.XmlReaderSettings.CheckCharacters%2A>|`true`.|
+|<xref:System.Xml.XmlReaderSettings.CloseInput%2A>|`false`.|
```
Comment (omajid) [inline xml/System.Xml/XmlReaderSettings.xml:126]: I looked at the online docs, and this property seems to exist even for older versions of .NET Framework: https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmlreadersettings.closeinput?view=netframework-3.0. 
I don't actually have a way to verify this, though.

As for the value, I used the default value defined in .NET source code: https://github.com/dotnet/runtime/blob/b4a1fa2293f31aa694e5a433aca676c29b50de81/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs#L419

**Comment by omajid** [comment]
cc @adegeo

**Review by adegeo — APPROVED** [review]
I checked in the framework 4.8 code and it's defaulting to false.

**Comment by adegeo** [comment]
@gewarren Hrmmm I don't perms here anymore (or never did, I don't know which lol)

**Pull request merged**